test/junit/scala/collection/mutable/OpenHashMapTest.scala

```diff
@@ -0,0 +1,42 @@
+package scala.collection.mutable
+
+import org.junit.Test
+import org.junit.Assert._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [[OpenHashMap]]. */
+@RunWith(classOf[JUnit4])
+class OpenHashMapTest {
+  /** Test that an [[OpenHashMap]] correctly maintains its internal `deleted` count. */
+  @Test
+  def maintainsDeletedCount {
+    import scala.reflect.runtime.{universe => ru}
```

Comment:
This import isn't needed if you're using Java reflection.  If you want to write a Scala reflection version, can you put it in a second method that is entirely commented out, and just leave this one as a clean Java-reflection version?